filebeat/reader/multiline/multiline_test.go

```diff
@@ -150,6 +151,25 @@ func TestMultilineBeforeNegateOKWithEmptyLine(t *testing.T) {
 	)
 }
 
+func TestMultilineAfterTruncated(t *testing.T) {
```

Comment:
Would be nice to have a test for a multiline event that also gets truncated and then check for both flags.

Reply:
Done.